jackal/uploader/queue.go

```diff
@@ -141,6 +140,10 @@ func (q *Queue) UpdateGecko() error {
 	q.jklPrice = priceResp.JackalPrice.USDPrice
 	log.Printf("updated gecko with %f", q.jklPrice)
 
+	err = resp.Body.Close()
```

Comment:
don't remove the defer, just replace with 

```
//nolint:errcheck
defer resp.Body.Close()
```

Reply:
oh lmao can we just do that